docs/downstream-integrations/api/authentication.md

```diff
@@ -24,22 +24,133 @@ The AE token must be passed in the HTTP `Authorization` (case insensitive) reque
 Authorization: AE <ae-token>
 ```
 
-The AE token is generated on the server when a user has successfully signed in with Collab.Land. The AE token is valid for 1 hour after which it expires and will need to be refreshed.
+The AE token is generated on the server when a user has successfully signed in with Collab.Land. The AE token is valid for 1 hour after which it expires and will need to be refreshed. -->
 
 ## Login with Collab.Land (LWC)
 
-The LWC flow process offers a way for client applications to authenticate their users through Collab.Land. Applications registered with Collab.Land can redirect users to `https://login.collab.land` to authenticate themselves with Collab.Land and grant permissions to the client application for requested resources.
+The Collab.Land API OAuth2 flow provides a secure and standardized method for client applications to authenticate users and access resources on Collab.Land. The flow follows the OAuth2 protocol, allowing applications to obtain an access token that can be used to interact with the Collab.Land API on behalf of the authenticated user.
+
+## Register your OAuth2 Client Application
+
+To begin using the Collab.Land API OAuth2 flow, you must first register your client application via the [Collab.Land Developer Portal](https://dev-portal.collab.land/). After the registration process, you will receive the following credentials:
+
+- **Client ID**: A unique identifier for your client application.
+- **Client Secret**: A confidential string used to authenticate your client application with Collab.Land.
+- **API Key**: A secret key used to authenticate your client application with Collab.Land.
+
+These credentials are essential for initiating the OAuth2 flow and obtaining access tokens.
+
+## Understand Scopes
+
+Collab.Land API uses scopes to define the level of access and permissions requested by the client application. Scopes determine the resources and operations that the access token can be used for. It is crucial to understand the available scopes and select the appropriate ones based on your application's requirements.
+
+The following scopes are available:
+
+**account:read**
+
+- Read all of your wallet address you connected with collab.land and balances of them.
+- Read basic user information including user name, id , pfp.
+- See communities you are in and administrating
+
+**access-control:read**
+Read user roles in a certain community
+
+**asset-ownership:read**
+Check ownership for the given account for the given token assets
+
+**gm-pass:read**
+Read gmPASS address and DID info
+
+**read-vcs**
```

Comment:
Let's refine the naming convention of scopes to make them easy to understand and consistent.

Reply:
Yeah, we'll ship this in the meantime and make the updates when its refined